docs/gro-data-key-concepts.md

```diff
@@ -0,0 +1,150 @@
+# Gro Data: Key Concepts
+
+1. [Gro Ontology](#gro-ontology)
+2. [Data Series Definition](#data-series-definition)
+3. [Data Point Definition](#data-point-definition)
+
+## Gro Ontology
+Gro has brought the world of agricultural data into a single ontology. This ontology classifies all data into a limited set of entities. 
+
+The entity types around which the data is organized in Gro are:
+
+items -- what the data is about (e.g., Rice, Sugar, Rainfall, etc.).\

```

Comment:
Remove slashes from the ends of these lines and add bullets to the beginnings of the lines

Reply:
Done